src/components/MapboxMap.tsx

```diff
@@ -169,26 +179,49 @@ const MapboxMap = ({
       new mapboxgl.Marker(bayBridgeEl)
         .setLngLat([-122.3778, 37.7983])
         .addTo(map);
+
+      setMapLoaded(true);
```

Comment:
to avoid race condition where useEffect runs before map.on("load") which prevented scrolling to correct school in list on back-button refresh